aspnetcore/performance/diagnostic-tools.md

```diff
@@ -11,21 +11,25 @@ uid: performance/diagnostic-tools
 
 By [Mike Rousos](https://github.com/mjrousos)
 
-This topic lists tools for diagnosing performance issues in ASP.NET Core.
+This article lists tools for diagnosing performance issues in ASP.NET Core.
 
 ## Visual Studio Diagnostic Tools
 
-The [profiling and diagnostic tools](https://docs.microsoft.com/visualstudio/profiling) built into Visual Studio 2017 are a great place to start investigating performance issues since they're powerful and easy to use from the Visual Studio development environment. The tooling allows analysis of CPU usage, memory usage, and performance events in ASP.NET Core applications.
+The [profiling and diagnostic tools](/visualstudio/profiling) built into Visual Studio are a good place to start investigating performance issues. These tools are powerful and convenient to use from the Visual Studio development environment. The tooling allows analysis of CPU usage, memory usage, and performance events in ASP.NET Core apps.
 
-More information is available in [Visual Studio documentation](https://docs.microsoft.com/visualstudio/profiling/profiling-overview).
+More information is available in [Visual Studio documentation](/visualstudio/profiling/profiling-overview).
 
 ## Application Insights
 
-[Application Insights](https://docs.microsoft.com/azure/application-insights/app-insights-overview) can provide in-depth performance data for your application. Application Insights automatically collects data on response rates, failure rates, dependency response times, and more. You can also log custom events and metrics specific to your app. 
+[Application Insights](/azure/application-insights/app-insights-overview) provides in-depth performance data for your app. Application Insight automatically collects data on response rates, failure rates, dependency response times, and more. Application Insights supports logging custom events and metrics specific to your app.
 
-When there's a performance issue in production (or even in development or staging environments), Application Insights data is the first place to look to understand what went wrong. Although Application Insights works great in Azure, it can also be used to gather telemetry while running locally from [Visual Studio](https://docs.microsoft.com/azure/application-insights/app-insights-visual-studio) or in other hosting environments. 
+Application Insights is one of the best tools to find performance issues:
```

Comment:
I feel like this doesn't relate to the bullets below it very well. Maybe rephrase this lead in to be something like "Application Insights can be used in many different environments:".